recipes-core/everest-chargebyte/everest-chargebyte_git.bb

```diff
@@ -25,6 +26,20 @@ EXTRA_OECMAKE += " \
     -Deverest-chargebyte_USE_PYTHON_VENV=OFF \
 "
 
+EVEREST_EXCLUDE_MODULES ?= ""
+
+EVEREST_EXCLUDE_MODULES:chargesom ?= " \
```

Comment:
It is simple `MACHINE`-specific override.
To be honest, I don't get your point here. But I assume that you want me to change it to weak assignments, which is indeed a good proposal since this can then be overwritten with bbappends (which is currently not the case AFAIK).